### ISSUE TITLE
Search page: Basic improvements.

### ISSUE THREAD

**Issue by jasmussen** [WordPress/wporg-main-2022#216]
Let's improve the [search page](https://wordpress.org/search/jetpack/):

<img width="1259" alt="Screenshot 2023-03-21 at 10 51 47 PM" src="https://user-images.githubusercontent.com/1204802/226575611-8c33d485-6801-4420-b590-ce0a4c87aa3e.png">

* [ ] The search header does not need to be 300px high, and doesn't need the subtext. Let's try a blue breadcrumb bar with just the "Search" subheading.
* [ ] Omit the register / login bar. In general, let's omit this on every single page we can, and show it only in the cases where it's relevant. On the pages where we do include the login bar, let's use the same bg color for the login bar, as the main navigation bar background, or integrate the login link in the prose.
* [ ] "Learn" and  "Forums" highlighted in menu, which doesn't seem relevant to search. Let's not highlight anything.

It appears that the search results use an embedded Google search. This comes with a couple of issues, design-wise:

* It becomes a separate search box instead of being unified with what exists in the header. Main problem with that is the disconnect with the search box in the header, which even if it kicked off the search, on the results page becomes inert and collapsed. There are a few ways we an thread this, designwise — "moving" the search box, unifying on the one in the header with it staying open, etc.  
* This duplicated search box is awkwardly balanced, low contrast, somewhat small in terms of emphasis (It's the most important part of the page).
* The search icon is reversed and different from the one in the header.
* The search tabs provided are not in sync with the header verbiage ("Support Forums" instead of "Forums", "Support Docs" instead of "Documentation", separate "Developer Docs", etc)
* The formatting of the results leaves a great deal to be desired. Depending on what's possible here, I'd love to provide a mockup.

What options do we have for designing the Google search results? And if our options are limited, are there alternatives we can explore?

**Comment by StevenDufresne** [comment]
> show it only in the cases where it's relevant

Can we come up with a definition of "relevant" before we start doing this? It isn't clear to me what that would mean.

> What options do we have for designing the Google search results? And if our options are limited, are there alternatives we can explore?

@ryelle Would have a better sense of what we can do with the search result.

Additionally, I think @pkevan has started to look at options for removing our dependency on google search results entirely.

**Comment by jasmussen** [comment]
> Omit the register / login bar. In general, let's omit this on every single page we can, and show it only in the cases where it's relevant. On the pages where we do include the login bar, let's use the same bg color for the login bar, as the main navigation bar background, or integrate the login link in the prose.

Yes, definitely. I opened https://github.com/WordPress/wporg-mu-plugins/issues/355 (was that the right place?) which may be a better place to open the conversation. That's also the reason for the phrasing, which perhaps could've been more verbose: my personal instinct is to never ever show the adminbar, on any site, unless you're logged in, and have login links exist elsewhere. 

However I also assume that there's been very good reasons for why it _was_ shown in some cases. I want to make sure we don't regress those cases, miss the history of this, or just move cheese for no good reason. For example if our contributors rely on the register/login buttons on the Make sites and use them commonly, we should definitely _keep_ the adminbar there, at least for until those sites can be refreshed separately at which point we can revisit those flows.

> has started to look at options for removing our dependency on google search results entirely.

Sounds good to me. My experience there is limited. I know that Jetpack opens search results in a somewhat customizable modal — that _might_ work well in keeping the search mostly global (invoked from search in the header), and could reduce (though not eliminate probably) our dependancy on having a separate search page.

**Comment by ryelle** [comment]
> Omit the register / login bar.

This is covered by https://github.com/WordPress/wporg-mu-plugins/pull/383

> "Learn" and "Forums" highlighted in menu, which doesn't seem relevant to search. Let's not highlight anything.

It's a little strange but I guess it considers the search results to be part of the "Forums" section. I've added a note to the section-highlighting issue to address that too: https://github.com/WordPress/wporg-mu-plugins/issues/370#issuecomment-1496632841

> The search tabs provided are not in sync with the header verbiage ("Support Forums" instead of "Forums", "Support Docs" instead of "Documentation", separate "Developer Docs", etc)

Unfortunately it looks like we can't change that.

I've made a few changes to the embed directly (so those are live already) and a few more at https://github.com/WordPress/wporg-main-2022/pull/248.
